openpype/tools/loader/widgets.py

```diff
@@ -567,12 +567,12 @@ def on_context_menu(self, point):
 
             # Trigger
             project_name = self.dbcon.active_project()
-            subset_names_by_version_id = collections.defaultdict(set)
+            subset_name_by_version_id = dict()
             for item in items:
                 version_id = item["version_document"]["_id"]
-                subset_names_by_version_id[version_id].add(item["subset"])
+                subset_name_by_version_id[version_id] = item["subset"]
```

Comment:
A version id would never have multiple subsets - so storing it as a set of subsets per version id didn't make much sense.